README.md

```diff
@@ -20,21 +20,17 @@ Projects have the flexibility to operate pNetwork v4 either as a standalone serv
 - **Global pTokens**: Ensures that users receive the same pToken when bridging across chains, maintaining a seamless and reliable user experience.
 - **Delegated Gas Costs**: Users are now responsible for executing transactions in the destination chain, giving them more control over their transactions and ensuring transparency in gas cost management.
 
-## Technical aspects
+## Monorepo structure
 
-pNetwork v4 introduces new on-chain entities:
+This morepo holds all the on-chain code needed for the protocol to work. In particular there are three folder of relevance:
 
-- **pNetwork Adapter contract:** needed to facilitate the wrapping and unwrapping of the ERC20 token. This would be the entry point for UIs.
-- **Lockbox:** vault where the collateral is deposited/released when wrapping/unwrapping ERC20 tokens.
-- **pNetwork fee contract:** responsible for holding all the node operator fees.
-- **pTokensV2:** the wrapped representation of the ERC20 token on the underlying chain.
-- **pNetwork Authorization Module (PAM):** verifies that the operation performed on the destination chain has been authorized by pNetwork Authorization System.
+- **solidity:** keeps all the EVM contract code, relative tests and deployment scripts
+- **cpp:** keeps all the EOSIO contract code, relative tests and deployment scripts
```

Comment:
Shouldn't this be `ANTELOPE contract code` instead of EOSIO?

Reply:
you're right, I'm still in 2013 :sweat_smile: